Packs/Illumio/IllumioCore/IllumioCore.py

```diff
@@ -0,0 +1,95 @@
+import demistomock as demisto
+import urllib3
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from illumio import PolicyComputeEngine
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+ERRORS = {
+    'INVALID_ORG_ID': "{} is an invalid value. Organization ID must be a non-zero and positive numeric value.",
+    'NO_RECORDS': 'No Record Found.',
+    'MISSING_REQUIRED_PARAM': "{} is a required parameter. Please provide correct value.",
+    'INVALID_PORT_NUMBER': '{} is an invalid value for port. Value must be in 1 to 65535.',
+    'INVALID_SINGLE_SELECT_PARAM': '{} is an invalid value for {}. Possible values are: {}.',
+    'INVALID_MULTI_SELECT_PARAM': "Invalid value for {}. Possible comma separated values are {}."
+}
+
+''' CLIENT CLASS '''
+
+
+class IllumioClient:
```

Comment:
Following conventions mentioned [here](https://xsoar.pan.dev/docs/integrations/code-conventions)

Reply:
The convention of using a `Client` class is predicated on the assumptions that

- it will inherit from `BaseClient` (which we're not doing here)
- it will reduce duplication and context passing through function params (which is mostly unnecessary since the `PolicyComputeEngine` class already handles this)

I don't see a legitimate case for wrapping the client in this outer shell.

Reply:
Yeah, but this is the convention we've been usually following in other integrations as well, as we need to make a call through the command function to our baseclient in this case IllumioClient.

Reply:
The `PolicyComputeEngine` class already behaves the same way that the docs suggest a `Client` wrapper should. There's no need to add an extra wrapper around it unless the `IllumioClient` class is providing some additional functionality.

Right now, it's just used as a passthrough - we pass the client instance to the handler functions which then call, e.g.

```
client.pce.ip_lists.get()
```

Why is this better than instantiating the `PolicyComputeEngine` instance in main directly and passing it instead?

Reply:
Totally agree with you as instantiating the `PolicyComputeEngine` class directly will save the redundancy, but we need to make sure that there shouldn't be a problem during the certification process of XSOAR, as the best practices suggest to have a client class.

Reply:
There are a number of packs that don't include a client, so I don't think it will cause an issue during the certification process:

https://github.com/demisto/content/blob/master/Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py
https://github.com/demisto/content/blob/master/Packs/ActiveMQ/Integrations/ActiveMQ/ActiveMQ.py
https://github.com/demisto/content/blob/master/Packs/AnsibleAzure/Integrations/AnsibleAzure/AnsibleAzure.py

Reply:
Yes, we had put a query on slack as well, we can directly instantiate the `PolicyComputeEngine `class.

Reply:
I've made the necessary changes & now it will directly instantiate the `PCE `class.